dbus_lynx_distributor/dbus_lynx_distributor_service.py

```diff
@@ -84,19 +80,19 @@ def _config_getboolean(self, option, fallback):
         return self._config.getboolean(f'ftdi:{self._ftdi.serial_number}', option, fallback=fallback)
 
     def _update(self):
+        sleep(5)
```

Comment:
Why an extra sleep? Isn't the sleep already achieved by`GLib.timeout_add(1000, self._update)`?

Reply:
I recognized that the cpu utilization with a high polling rate was quite high, so I throttled it with this sleep and afterwards it was fine regarding this

Reply:
You replace the sleep in the other PR with 1s. If you want the timeout to be increased to 2s total, then we should change the value passed to `timeout_add`. I don't observe any high CPU load.

Reply:
ok, I overlooked that and removed the sleep